src/registry_schemas/example_data/schema_data.py

```diff
@@ -49,7 +49,28 @@
     'lastLedgerTimestamp': '2019-04-15T20:05:49.068272+00:00',
     'lastPreBobFilingTimestamp': '2019-04-15T20:05:49.068272+00:00',
     'legalName': 'legal name - CP1234567',
-    'legalType': 'CP'
+    'legalType': 'CP',
+    'state': 'ACTIVE',
+    'goodStanding': True,
+    'complianceWarnings': [
```

Comment:
As an example, what would an entity pending dissolution look like?

Reply:
Is this example valid? I see that the business has compliance warnings but is still in good standing.

How can the UI tell if the business is not in compliance?

Reply:
As per my understanding, an entity that is pending dissolution is Active. It moves to the historical state only if the dissolution is processed successfully by the filer.

Reply:
OK.

So how can the UI tell if the business is pending dissolution?

Reply:
Also, if there are compliance warnings, does that mean the business is not in compliance?

Reply:
Also, is a filing pending dissolution in good standing? I guess it depends on what kind of dissolution this is... Can we list out the various states please?

Re: https://github.com/bcgov/business-filings-ui/blob/main/src/enums/dissolutionTypesNames.ts

Reply:
4. When is a business not in good standing?

Reply:
A business 

> 3. If there are compliance warnings, does that mean the business is not in compliance?

Yes, compliance warnings indicate that the business is not in compliance

Reply:
> 2. So how can the UI tell if the business is pending dissolution?

yes, this is something we have to figure out unless @thorwolpert or @lekshmimallika-aot already see how to deal with this.  Do we want to add a property like the following to the business?

```
pendingStateChange: {
   'filing': 'https://LEGAL-API-HOST/api/v2/businesses/IDENTIFIER/filings/FILING_ID'
   'filingType': 'dissolution',
   'toState': 'HISTORICAL'
}
```

Maybe only the filing link is required.

Reply:
> 5. When is a business not in good standing?

There are some rules for this. One of the examples given in the ticket is 'This applies to all companies that have an annual report that has been outstanding for more than 1 year, 2 months and 1 day'

Reply:
> 4. Is a filing pending dissolution in good standing? I guess it depends on what kind of dissolution this is... Can we list out the various states please?
> 
> Re: https://github.com/bcgov/business-filings-ui/blob/main/src/enums/dissolutionTypesNames.ts

I am not sure whether I understand it correctly, If you are referring to the dissolution initiated due to the failure to file annual reports and failure to transition you will get goodStanding as false in the response. I am not sure about some of the types like DISS_LLP_REG, DISS_CONVERTED and DISS_OTHER. Linda/Thor would be able to ans this more specifically.

Reply:
I'm moving this discussion to https://chat.developer.gov.bc.ca/channel/Dzh6CW3WTkDieEqe7?msg=vn7xCEcFtPY7b3QDh